modules/FIC_Github.py

```diff
@@ -114,3 +120,172 @@ def push(self):
     def revert_modified_files(self):
         from modules.config import CHANGELOG_JSON_PATH, CHANGELOG_MD_PATH, CHANGELOG_REPO_PATH
         return self.repo.git.checkout([CHANGELOG_JSON_PATH, CHANGELOG_MD_PATH, CHANGELOG_REPO_PATH])
+
+    def get_repo_url(self):
+        return self.repo_data.svn_url
+
+    def _repo_team(self):
+        self.team_name = json.load(self.load(None, "repositories.json")).get("Github").get(self.repo_name).get("team")
+
+    def _repo_files(self):
+        self.folders_to_check = json.load(self.load(None, "repositories.json")).get("Github").get(self.repo_name).get("configuration").get("folders-to-check")
+
+    def _extract_repo_type(self):
+        self.repo_type = json.load(self.load(None, "repositories.json")).get("Github").get(self.repo_name).get("configuration").get("type")
+
+    def _local_version(self):
+        self.local_version = json.load(self.load(CHANGELOG_REPO_PATH, self.repo_name.lower() + ".json")).get("0").get("last_release").get("version")
+
+    def _get_release(self):
+        self.release_version = [tag for tag in self.repo_data.tags(number=1)][0].name
+
+    def _get_version_path(self):
+        self.version_path = json.load(self.load(None, "repositories.json")).get("Github").get(self.repo_name).get("configuration").get("version-path")
+
+    def _build_puppet_version(self):
+        self._get_version_path()
+        for requirements in requests.get(self.version_path).text.split():
+            if self.repo_name in requirements:
+                self.build_puppet_version = re.split("\\b==\\b", requirements)[-1]
+                return True
+
+    def _compare_versions(self):
+        if self.build_puppet_version == self.release_version:
+            if self.release_version != self.local_version:
+                return True
```

Comment:
this can be redefined as:
```Python
def _compare_version(self):
    if (self.build_puppet_version == self.release_version) and (self.release_version != self.local_version):
        return True
    else:
        return False
```

Reply:
It was done in: https://github.com/mozilla-releng/firefox-infra-changelog/pull/448/commits/16f80b00b7463d786334be5bc955e8dfac03b020